test/Interop/Cxx/members/emit-method-from-called-method.swift

```diff
@@ -0,0 +1,14 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
```

Comment:
Could you rename the .swift tests to match the name of the header file? This would match our existing naming pattern.

`<header-file>-irgen.swift` & `<header-file>-execution.swift`

So this one would be `method-calls-method-execution.swift`.

Reply:
Done. Thanks!